index.js

```diff
@@ -603,4 +603,18 @@ export const mentorshipTeam = [
 			},
 		],
 	},
+	{
+		name: 'Vy Nguyen',
+		title: 'Student @ Catholic University of America',
+		hobbies: 'Music, cooking, travel',
+		avatar:
+			'https://drive.google.com/file/d/1OxDKkXQtmnqiWUw8SZg50gYTtSL8pk4K/view?usp=sharing',
```

Comment:
let's use this new URL for reliability: `https://res.cloudinary.com/khoa165/image/upload/v1711778695/viettech/vynguyen.jpg`